Client/Frontend/Settings/SettingsContentViewController.swift

```diff
@@ -0,0 +1,180 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import SnapKit
+import UIKit
+import WebKit
+
+let DefaultTimeoutTimeInterval = 10.0 // Seconds.  We'll want some telemetry on load times in the wild.
+
+/**
+ * A controller that manages a single web view and provides a way for
+ * the user to navigate back to Settings.
+ */
+class SettingsContentViewController: UIViewController, WKNavigationDelegate {
```

Comment:
Can we share some implementation between this and FxAContentDelegate, which you cribbed this from?